tests/functional/requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/openstack-charmers/zaza.git#egg=zaza
+git+https://github.com/rgildein/zaza.git@chore/no-ref/fix-action-object#egg=zaza
```

Comment:
Could we leave a comment or issue link to indicate when we can use official repository again? thanks!

Reply:
Check description, "Blocked until https://github.com/openstack-charmers/zaza/pull/655 is merged." I do not want to merge this until that PR is not merged, what should be soon.